Dockerfile

```diff
@@ -28,11 +28,33 @@ FROM base as development
 
 CMD [ "poetry", "run", "flask", "run", "--host=0.0.0.0", "--port=5000", "--debugger"] 
 
+# testing stage
+FROM base as test
+RUN pip install pytest
```

Comment:
This step should be (and indeed from testing is!) unnecessary since we're installing pytest with poetry. Note that similarly the `RUN poetry add pytest --dev` step below is also unnecessary if we instead just run `poetry install` which will install that if it's tracked in your `pyproject.toml` and `poetry.lock` files (which it is). That way, if we were to add more dev-only packages they'd be automatically installed in the test image, without having to modify this further

Reply:
updated